src/Translator.php

```diff
@@ -0,0 +1,512 @@
+            $this->exec('tx push -s', $modulePath);
+        }
+    }
+
+    /**
+     * Commit changes for all modules
+     */
+    private function gitCommitPushAndPullRequest(): void
+    {
+        $this->log('Committing translations to git');
+        foreach ($this->modulePaths as $modulePath) {
+            // Get endpoint
+            $remote = $this->exec('git config --get remote.origin.url', $modulePath);
+            $remote = str_replace('git@github.com:', 'https://github.com/', $remote);
+            if (!preg_match('#^https://github.com/([^/]+)/(.+?)$#', $remote, $matches)) {
+                throw new Exception("Invalid git remote $remote");
+            }
+            array_shift($matches);
+            list($account, $repo) = $matches;
+            $repo = preg_replace('#\.git$#', '', $repo);
+            $endpoint = "https://api.github.com/repos/$account/$repo/pulls";
+
+            // Add remote
+            if (!in_array('tx-ccs', explode("\n", $this->exec('git remote', $modulePath)))) {
+                $this->exec("git remote add tx-ccs git@github.com:creative-commoners/$repo.git", $modulePath);
+            }
+
+            // Git add all changes
+            $jsPath = $this->getJSLangDirectories($modulePath);
+            $langPath = $this->getLangDirectory($modulePath);
+            foreach (array_merge((array) $jsPath, (array) $langPath) as $path) {
+                if (is_dir($path)) {
+                    $this->exec("git add $path/*", $modulePath);
+                }
+            }
+            $this->exec("git add .tx/config", $modulePath);
+
+            // Create new branch
+            $currentBranch = $this->exec('git rev-parse --abbrev-ref HEAD', $modulePath);
+            $time = time();
+            $branch = "pulls/$currentBranch/tx-$time";
+            $this->exec("git checkout -b $branch", $modulePath);
+
+            // Commit changes
+            $title = 'ENH Update translations';
+            $this->exec("git commit -m \"$title\"", $modulePath);
+
+            if ($this->isDevMode) {
+                $this->log('Not pushing changes or creating pull-request because TX_DEV_MODE is enabled');
+                return;
+            }
+
+            // Push changes to creative-commoners
+            $this->exec("git push --set-upstream tx-ccs $branch", $modulePath);
+
+            // Create pull-request via github api
+            $body = 'Automated translations update generated using silverstripe/tx-translator';
+            $client = new Client();
+            $response = $client->request('POST', $endpoint, [
+                'headers' => [
+                    "Accept" =>  "application/vnd.github.v3+json",
+                    "Authorization" => "token {$this->githubToken}"
+                ],
+                'body' => json_encode((object) [
+                    "title" => "$title",
+                    "body" => $body,
+                    "head" => "creative-commoners:$branch",
```

Comment:
This is intentionally hardcoded to creative-commoners. It's only intended to be used by Silverstripe CMS Squad, similar to how cow is now